styles/components/_playerPanel.scss

```diff
@@ -0,0 +1,64 @@
+.player-panel {
+  display: flex;
+  flex-grow: 1 1 1;
+  min-height: 40vh;
+  width: 100vw;
+
+  &__board {
+    width: 60vw;
+    position: relative;
+    padding: 0.5rem;
+    border-right: 1px solid black;
+  }
+
+  &__details {
+    display: inline-block;
+    margin-left: 1rem;
+  }
+
+  &__name {
+    font-size: 1.7rem;
+    font-weight: 700;
+  }
+
+  &__time {
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem;
+    display: none;
```

Comment:
I understand this part is not ready?

Reply:
It is ready. by default time remaining will not be displayed, will be displayed only, when timer will be running.